app/src/js/utils/table-config/reconciliation-reports.js

```diff
@@ -11,7 +11,7 @@ export const tableColumns = ({ dispatch, isGranules, query }) => ([
     Header: 'Name',
     accessor: 'name',
     Cell: ({ cell: { value }, row: { original: { type } } }) => { // eslint-disable-line react/prop-types
-      const link = (location) => ({ pathname: `/reconciliation-reports/report/${encodeURIComponent(value)}`, search: getPersistentQueryParams(location) });
+      const link = (location) => ({ pathname: `/reconciliation-reports/report/${value}`, search: getPersistentQueryParams(location) });
```

Comment:
Looks like a bad merge here?  I think the encodedURI was Jenny's fix for unreadable filenames.   And that's what failed your build it looks like.

Reply:
woops! will fix